modules/ocf_mesos/manifests/secrets.pp

```diff
@@ -31,6 +31,16 @@
       purge     => true,
       force     => true,
       show_diff => false;
+
+    # Create a couple scratch directories for sourcegraph to use for temporary
+    # data storage
+    #
+    # This isn't great, but I think it's better than creating a whole new repo
+    # for sourcegraph and this temporary directory only stores session
+    # information and cloned repos anyway
+    ['/opt/share/docker/sourcegraph', '/opt/share/docker/sourcegraph/redis']:
```

Comment:
does souregraph run its own redis in the container? can it use gridlock?

Reply:
It does run its own redis, we could maybe use gridlock, but sourcegraph doesn't seem to support redis authentication from what I can tell: https://about.sourcegraph.com/docs/config/external-database/#using-your-own-redis-server-with-sourcegraph (and it only stores ephemeral data)

Reply:
I see. Probably not worth the effort, then.